### PR TITLE
implementation for DoubleTap action in teswiz

### PR DESCRIPTION
**Pull request by Darshan3S**
This PR covers the Implementation for DoubleTap Action in teswiz framework 
-Added DoubleTap action for driver
-Added Scenario to test the functionality in VodQA application 
-Added methods required for the same

### CODE REVIEW

File: src/test/java/com/znsio/teswiz/screen/android/vodqa/VodqaScreenAndroid.java [hunk 1]
```diff
@@ -169,4 +173,18 @@ public VodqaScreen scrollVerticallyByPercentage(int fromPercentHeight, int toPer
         visually.checkWindow(SCREEN_NAME, "Screen scrolled down");
         return this;
     }
+
+    @Override
+    public VodqaScreen doubleTapOnElement() {
+        LOGGER.info("Performing Double tap on element in Double tap screen");
+        driver.waitTillElementIsVisible(byDoubleTapScreenXpath).click();
```
Comment (snehith07) [inline src/test/java/com/znsio/teswiz/screen/android/vodqa/VodqaScreenAndroid.java:180]: visual validations are missing

File: src/test/java/com/znsio/teswiz/screen/android/vodqa/VodqaScreenAndroid.java [hunk 2]
```diff
@@ -169,4 +173,18 @@ public VodqaScreen scrollVerticallyByPercentage(int fromPercentHeight, int toPer
...
+        driver.doubleTap(driver.waitTillElementIsVisible(byDoubleTapElementXpath));
+        return this;
+    }
+
+    @Override
+    public boolean isDoubleTapSuccessful() {
+        LOGGER.info("Checking if double tap on element is successful");
+        return driver.isElementPresent(byDoubleTapSuccessfulXpath);
```
Comment (snehith07) [inline src/test/java/com/znsio/teswiz/screen/android/vodqa/VodqaScreenAndroid.java:188]: add visual validations before and after the double tap action

File: src/test/java/com/znsio/teswiz/screen/android/vodqa/VodqaScreenAndroid.java (hunk 1 above)
Comment (Darshan3S) [reply src/test/java/com/znsio/teswiz/screen/android/vodqa/VodqaScreenAndroid.java:180]: Added visual validations

File: src/test/java/com/znsio/teswiz/screen/android/vodqa/VodqaScreenAndroid.java (hunk 2 above)
Comment (Darshan3S) [reply src/test/java/com/znsio/teswiz/screen/android/vodqa/VodqaScreenAndroid.java:191]: Added visual validations

File: src/main/java/com/znsio/teswiz/runner/Driver.java [hunk 3]
```diff
@@ -16,24 +16,16 @@
...
 import org.apache.commons.lang3.NotImplementedException;
 import org.apache.log4j.Logger;
-import org.openqa.selenium.By;
-import org.openqa.selenium.Dimension;
-import org.openqa.selenium.JavascriptExecutor;
-import org.openqa.selenium.NoSuchElementException;
-import org.openqa.selenium.Point;
-import org.openqa.selenium.WebDriver;
-import org.openqa.selenium.WebElement;
+import org.openqa.selenium.*;
```
Comment (anandbagmar) [inline src/main/java/com/znsio/teswiz/runner/Driver.java:22]: use single class imports

Comment (Darshan3S) [reply src/main/java/com/znsio/teswiz/runner/Driver.java]: updated with single class imports

**Pull request merged**